backend/src/index.ts

```diff
@@ -19,8 +19,10 @@ async function main() {
   })
 
   // start DHT hyperswarm when DHT_TOPIC is set in .env
-  if (CONFIG.DHT_TOPIC) {
-    await startDHT(CONFIG.DHT_TOPIC) // con,
+  if (CONFIG.FEDERATION_DHT_TOPIC) {
+    // eslint-disable-next-line no-console
+    console.log(`Federation active on ${CONFIG.FEDERATION_DHT_TOPIC}`)
```

Comment:
Here you can produce the seed if you think thats important - `starting Federation with seed ...`

Reply:
This strange behaviour in the keypair with or without seed doesn't appear on using a normal Buffer instead of the sodium malloced one. Don't know why, but with the current solution both key looks quite good or as cryptical as expected.

Reply:
...but i add the seed config-info in this log-output